config/initializers/validators/certificate_validator.rb

```diff
@@ -8,8 +8,8 @@ def self.certificate_is_new(value)
     validate :value, :certificate_is_new, on: :create, if: :value_present?
   end
 
```

Comment:
We shouldn't be monkey patching ActiveRecord::Base with non-general behaviour. Instead we can either use a [ActiveModel::Validator](https://api.rubyonrails.org/classes/ActiveModel/Validator.html) or a ActiveRecord::Concern

Reply:
Yup I agree. I will tackle this in a separate PR.